src/marks/geo.js

```diff
@@ -34,6 +34,7 @@ export class Geo extends Mark {
     const {geometry: G, r: R} = channels;
     const path = geoPath(context.projection);
     const {r} = this;
+    if (this.r != null && negative(this.r)) return;
```

Comment:
This is equivalent:

```js
    if (negative(r)) return;
```

but per the top comment, I think we want something like this:

```js
    if (negative(r)) index = [];
```